README.md

```diff
@@ -39,7 +39,7 @@ Both projects are whitelabelled with different [build configs](build-configs/REA
 There are also separate sub-projects used by both projects:
 
 - **[translations](translations/README.md)**: Translated texts and utilities to manage them
-- **[api-client](api-client/README.md)**: Implementation of the endpoints (v3) of the integreat cms
+- **[shared](shared/README.md)**: Implementation of the endpoints (v3) of the integreat cms
```

Comment:
Maybe you can adjust the description a little/add that it also includes shared things (utils, constants, ...)...